src/uwtools/resources/rocoto.jsonschema

```diff
@@ -343,8 +375,7 @@
     "time": {
       "oneOf": [
         {
-          "pattern": "^-?([0-9]+|(#[^# ]+#))+$",
-          "type": "string"
+          "type": "integer"
```

Comment:
Here and in `nonNegativeTime`, the type of the accepted input has been changed to `integer` (and with a minimum value of 0 for `nonNegativeTime`) so that e.g. `3600` can be used in YAML.